cmd/config/cluster-density-v2/cluster-density-v2.yml

```diff
@@ -8,6 +8,14 @@ global:
         - conditionType: Ready
           metric: P99
           threshold: {{.POD_READY_THRESHOLD}}
+    - name: pprof
+      pprofInterval: 2m
+      pprofDirectory: pprof-data
+      pprofTargets:
+      - name: ovn-profile
+        namespace: "openshift-ovn-kubernetes"
+        labelSelector: {app: ovnkube-node}
+        url: http://localhost:29103/debug/pprof/profile?seconds=30
```

Comment:
Can we toggle via `pprof_needed=yes`, so we don't always default to collect

Reply:
good point, updated

Reply:
should we have all the ovn components vs just ovnk-node?

Reply:
added ocn-controller and ovnk-control plane